api/sql/schema/750$transfer.idle.execute.sql

```diff
@@ -160,16 +162,8 @@ BEGIN TRY
             [transfer].[partner] lp ON lp.partnerId = t.ledgerId
         LEFT JOIN
             [transfer].[partner] ip ON ip.partnerId = t.issuerId
-        OUTER APPLY (
-            SELECT TOP 1
-                udfDetails
-            FROM
-                [transfer].[event]
-            WHERE
-                transferId = t.transferId AND source = 'acquirer' AND state <> 'request'
-            ORDER BY
-                eventDateTime DESC
-        ) AS er
+        LEFT JOIN
+            [transfer].[event] er ON er.transferId = t.transferId AND er.source = 'acquirer' AND er.state = 'fail'
```

Comment:
We used `OUTER APPLY` because more than one row can match, which will generate multiple reversal rows for the same transfer.

Reply:
https://github.com/softwaregroup-bg/ut-transfer/blob/master/api/sql/schema/750%24transfer.push.failAcquirer.sql#L15

considering this procedure i don't think that it's possible to have more than 1 row with source = 'acquirer' and state = 'fail'